bucket/gimp.json

```diff
@@ -65,7 +65,13 @@
     ],
     "checkver": {
         "url": "https://www.gimp.org/downloads/",
-        "regex": "gimp-([\\d.]+)-setup(-\\d)?\\.exe",
-        "replace": "${1}${2}"
+        "regex": "gimp-(?<version>[\\d.]+)-setup(?<build>-\\d)?\\.exe",
+        "replace": "${version}${build}"
+    },
+    "autoupdate": {
+        "url": "https://download.gimp.org/mirror/pub/gimp/v$majorVersion.$minorVersion/windows/gimp-$version-setup$matchBuild.exe",
```

Comment:
Hey I feel like I made a mistake

This did not end up working for v2.10.24-3

For that I had to switch to https://download.gimp.org/mirror/pub/gimp/v$majorVersion.$minorVersion/windows/gimp-$majorVersion.$minorVersion.$patchVersion-setup$matchBuild.exe